src/firebase.js

```diff
@@ -0,0 +1,28 @@
+// import { initializeApp } from "firebase/app";
+// import { getFirestore } from 'firebase/firestore';
+
+
+// const firebaseConfig = {
+//     apiKey: "AIzaSyD8rhlUKMh9wP3uVuTD6UjHxJsADGDWAX8",
+//     authDomain: "metaverse-a531c.firebaseapp.com",
+//     projectId: "metaverse-a531c",
+//     storageBucket: "metaverse-a531c.appspot.com",
+//     messagingSenderId: "308442262072",
+//     appId: "1:308442262072:web:3f783e0d9a3b7bf98614cd"
+//   };
+  
+//   // Initialize Firebase
+//   const app = initializeApp(firebaseConfig);
+//   const db = getFirestore();
+
+import firebase from "firebase/compat/app";
+import "firebase/compat/firestore";
+
+firebase.initializeApp({
```

Comment:
What's the purpose of this file when you initialize firebase in index.js also?

Reply:
This is sloppiness by me sorry. I should have deleted it